### DISCUSSION TITLE
Docker image of this?

### DISCUSSION THREAD

**Discussion by DrewThomasson** [lukaszliniewicz/Pandrator#32] [Ideas]
For easy testing and such?

**Comment by lukaszliniewicz** [comment]
I hate Docker xd But it's a good suggestion, many people use it. Will look into it. In any case, I'll make a prepackaged version with all environments and dependencies. And once the TTS engine APIs and/or tools are installed, it's easy to update to the latest version of the app itself from the GUI or through a simple git pull.
